users/postgres/users.go

```diff
@@ -71,7 +71,7 @@ func (ur userRepository) UpdateUser(ctx context.Context, user users.User) error
 }
 
 func (ur userRepository) RetrieveByID(ctx context.Context, email string) (users.User, error) {
```

Comment:
If its `RetrieveByID` why param is `email` ?

Reply:
@nmarcetic because in Users service we use email as unique ID

Reply:
Yea because we didn't have `ID` but now when we introduce `ID`, this should be changed.